recipes/ozo/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.12)
```

Fairly certain 17 was released a bit earlier 

```suggestion
cmake_minimum_required(VERSION 3.8)
```